### PR TITLE
added GA

### PR DESCRIPTION
**Pull request by bossbossk20**
added Google Analytic using react-ga

### CODE REVIEW

**Comment by bossbossk20** [comment]
#61

**Comment by DSchau** [comment]
Deploy preview ready!

Built with commit c0c35161545f2cfd8df1b52c39925c9aaa837ee9

https://deploy-preview-64--violinist-oxygen-57835.netlify.com

File: tsconfig.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "moduleResolution": "node",
     "jsx": "react",
     "lib": ["dom", "es5", "es2015", "webworker"],
+    "allowSyntheticDefaultImports": true,
```
Comment (DSchau) [inline tsconfig.json]: This change breaks a few other things, I believe!

**Review by DSchau — CHANGES_REQUESTED** [review]
I'm open to the idea of using react-ga, but only if I'm getting (extra) value out of it vs. just the inline script

I hate to be a pain, but I think I'd prefer the inline script route in src/index.html (which is easier too).

Want to give that a shot?

**Review by DSchau — APPROVED** [review]

**Comment by DSchau** [comment]
@bossbossk20 could you "allow edits from maintainers" in the right  hand side of the PR?

I would like to use this script, otherwise this is perfect :)

```
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-102928446-1', 'auto');
ga('send', 'pageview');
</script>
```

**Comment by DSchau** [comment]
@bossbossk20 also sorry for the lack of direction, should've been way clearer up-front what I was hoping for! Thank you again :)

**Comment by bossbossk20** [comment]
I did it @DSchau

**Comment by bossbossk20** [comment]
hey

**Review by DSchau — CHANGES_REQUESTED** [review]
Why are there other file edits in this PR? Also react-ga is included in package.json

**Review by DSchau — APPROVED** [review]

**Pull request merged**